src/diffpy/utils/diffraction_objects.py

```diff
@@ -320,70 +375,15 @@ def _set_xarrays(self, xarray, xtype):
             self._all_arrays[:, 3] = xarray
             self._all_arrays[:, 1] = d_to_q(xarray)
             self._all_arrays[:, 2] = d_to_tth(xarray, self.wavelength)
+
+    def _set_min_max_xarray(self):
```

Comment:
`_set_arrays` is used to set `all_arrays` and I thought it would be more modular to have `_set_min_max_xarray` for determining min/max values of q, tth, d.